### PR TITLE
Upgrade storybook to latest version

### PR DESCRIPTION
**Pull request by micgro42**
This also switches to the new package-lock format

T301155

### CODE REVIEW

**Comment by micgro42** [comment]
Browser test failing:

sauceFirefox: |

File: .github/workflows/test-and-demo.yaml [hunk 1]
```diff
@@ -35,9 +35,9 @@ jobs:
           behavioral:
             - 'vue-components/**'
     - name: Set up node
-      uses: actions/setup-node@v1
+      uses: actions/setup-node@v2
```
Comment (lucaswerkmeister) [inline .github/workflows/test-and-demo.yaml:38]: Is it intentional that most of the other setup-node lines are still at v1 AFAICT?

**Review by lucaswerkmeister — COMMENTED** [review]
When I try this locally (with Node v16.13.2, npm v8.1.2), I get the following error on `npm run build:storybook` (after `npm ci`):

```
ERR! Error: Cannot find module 'react-dom/package.json'                                                                                                                                                                                                                                                                                                    
ERR! Require stack:                                                                                                                                                                                                                                                                                                                                        
ERR! - /home/luwe/git/wikit/docs/node_modules/@storybook/ui/noop.js                                                                                                                                                                                                                                                                                        
ERR!     at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)                                                                                                                                                                                                                                                                     
ERR!     at resolveFileName (/home/luwe/git/wikit/docs/node_modules/resolve-from/index.js:29:39)                                                                                                                                                                                                                                                           
ERR!     at resolveFrom (/home/luwe/git/wikit/docs/node_modules/resolve-from/index.js:43:9)                                                                                                                                                                                                                                                                
ERR!     at module.exports (/home/luwe/git/wikit/docs/node_modules/resolve-from/index.js:46:47)                                                                                                                                                                                                                                                            
ERR!     at Object.<anonymous> (/home/luwe/git/wikit/docs/node_modules/@storybook/ui/paths.js:18:24)                                                                                                                                                                                                                                                       
ERR!     at Module._compile (node:internal/modules/cjs/loader:1101:14)                                                                                                                                                                                                                                                                                     
ERR!     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
ERR!     at Module.load (node:internal/modules/cjs/loader:981:32)
ERR!     at Function.Module._load (node:internal/modules/cjs/loader:822:12)
ERR!     at Module.require (node:internal/modules/cjs/loader:1005:19)
ERR!     at require (node:internal/modules/cjs/helpers:102:18)
ERR!     at Object.<anonymous> (/home/luwe/git/wikit/docs/node_modules/@storybook/manager-webpack4/dist/cjs/presets/manager-preset.js:63:38)
ERR!     at Module._compile (node:internal/modules/cjs/loader:1101:14)
ERR!     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
ERR!     at Module.load (node:internal/modules/cjs/loader:981:32)
ERR!     at Function.Module._load (node:internal/modules/cjs/loader:822:12)
```

(And then a few more copies, but it seems to be the same error, `react-dom/package.json`.)

**Comment by lucaswerkmeister** [comment]
Weird, after removing the nested `node_modules/` dirs it worked. I guess `npm ci` isn’t fully clean under Lerna…?

**Review by lucaswerkmeister — APPROVED** [review]

**Comment by micgro42** [comment]
> Weird, after removing the nested `node_modules/` dirs it worked. I guess `npm ci` isn’t fully clean under Lerna…?

Yeah, lerna is strange and often leads to diffs in the `package-lock.json` the reasons to which are not always clear to me. But I often try run things only via lerna as that is the closest to what CI does.

**Pull request merged**